src/System.ComponentModel.TypeConverter/tests/MemberDescriptorTests.cs

```diff
@@ -10,6 +10,24 @@ namespace System.ComponentModel.Tests
     public class MemberDescriptorTests
     {
         [Fact]
+        public void CopiedMemberDescriptorEqualsItsSource()
+        {
+            var attributes = new Attribute[]
+            {
+                new CategoryAttribute("category"),
+                new DescriptionAttribute("description")
+            };
+            var firstDescriptor = new MockMemberDescriptor(nameof(MemberDescriptor), attributes);
+            var copiedDescriptor = new MockMemberDescriptor(firstDescriptor);
+
+            // call getters to ensure their backing fields aren't null
+            Assert.True(firstDescriptor.Category == copiedDescriptor.Category);
+            Assert.True(firstDescriptor.Description == copiedDescriptor.Description);
```

Comment:
Consider using `Assert.Equals(expected, actual)` as this gives a better error message if an assertion fails